src/Compilers/CSharp/Test/Syntax/Parsing/RefReadonlyTests.cs

```diff
@@ -425,9 +419,15 @@ void M(readonly ref int p)
                 // (4,12): error CS1002: ; expected
                 //     void M(readonly ref int p)
                 Diagnostic(ErrorCode.ERR_SemicolonExpected, "readonly").WithLocation(4, 12),
-                // (4,30): error CS1003: Syntax error, '(' expected
+                // (4,30): error CS1003: Syntax error, ',' expected
                 //     void M(readonly ref int p)
-                Diagnostic(ErrorCode.ERR_SyntaxError, ")").WithArguments("(", ")").WithLocation(4, 30));
+                Diagnostic(ErrorCode.ERR_SyntaxError, ")").WithArguments(",", ")").WithLocation(4, 30),
+                // (5,6): error CS1002: ; expected
```

Comment:
>// (5,6): error CS1002: ; expected [](http://example.com/codeflow?start=16&length=34)

Consider adding a syntax tree test for this scenario. I am assuming we used to parse parameter as a method and now we parse it as a field.